doc/tutorials/README.md

```diff
@@ -19,6 +19,7 @@ to participate in the global key database Elektra
 provides.
 
 - [Integration of your Application](application-integration.md)
+- [Hello, Elektra](hello-elektra.md)
```

Comment:
Should we put this first?

Reply:
I thought about that too. In my opinion it makes sense to read a little bit of introductory text – in the form of “Integration of your Application” –  before someone starts to write their first Elektra application.